src/FloatingActionButton/FloatingActionButton.spec.js

```diff
@@ -27,4 +29,26 @@ describe('<FloatingActionButton />', () => {
       assert.strictEqual(wrapper.state().hovered, false, 'should reset the state');
     });
   });
+
+  describe('style', () => {
+    it('should apply children style', () => {
+      const wrapper = mountWithContext(
```

Comment:
I read the docs of enzyme and it says:

> Shallow rendering is useful to constrain yourself to testing a component as a unit, and to ensure that your tests **aren't indirectly asserting** on behavior **of child components**

While for mount:

> Full DOM rendering is ideal for use cases where you have components **that may interact with DOM APIs**, or may require the full lifecycle in order to fully test the component (i.e., componentDidMount etc.)

I'm in doubt to use the ```mount``` instead of ```shallow``` because with ```shallowWithContext``` the test is still passed.
Thus, I don't have any other arguments to use ```mount```.